heron/storm/src/java/shade.conf

```diff
@@ -0,0 +1,4 @@
+rule com.google.** org.apache.storm.shaded.@0
+rule com.twitter.** org.apache.storm.shaded.@0
```

Comment:
@objmagic - the intention of com.twitter.heron.xxx is to hide implementation details. Nobody should access the com.twitter.heron.xxx api - since it is bound to change. Developers should stick with org.apache.storm.xxx or backtype.storm.xxx apis only.